sirepo/package_data/static/js/raydata.js

```diff
@@ -809,15 +817,37 @@ SIREPO.app.directive('scansTable', function() {
             };
 
             $scope.columnIsSortable = (column) => {
-                return column !== 'stop';
+                if (['stop'].includes(column)) {
+                    return false;
+                } else if ($scope.scans.length > 0 && typeof $scope.scans[0][column] === 'object') {
+                    return false;
+                } else {
+                    return true;
+                }
```

Comment:
It looks like the "sortable" value is getting confused when I sort on a field.

For example, I'm viewing CHX with the columns (sequence id, T_sample_, acquire period, T_yoke, detectors, samples). When I visit the page, it is sorted by start. Then I select "acquire period" and it sort that column, but I can no longer select the column again to reverse the sorting. The other columns are also unsortable at this point, except for "start" and "sample".

Reply:
[b98f2e9](https://github.com/radiasoft/sirepo/pull/7163/commits/b98f2e99939da3aafba92716dbfd92e31ed4d63a)